cmd/antrea-agent/agent.go

```diff
@@ -629,6 +631,23 @@ func run(o *Options) error {
 			o.enableAntreaProxy)
 	}
 
+	var packetSamplingController *packetsampling.Controller
+	if features.DefaultFeatureGate.Enabled(features.PacketSampling) {
+		packetSamplingController = packetsampling.NewPacketSamplingController(
+			k8sClient,
+			crdClient,
+			serviceInformer,
+			endpointsInformer,
+			packetSamplingInformer,
+			ofClient,
+			ifaceStore,
+			networkConfig,
```

Comment:
I don't find its usage

Reply:
https://github.com/antrea-io/antrea/blob/c9f5102f49804fbd963332fdd99a1978b9cefc7d/cmd/antrea-agent/agent.go#L792

Reply:
I mean `networkConfig`

Reply:
ok, will remove this. Thanks